stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/viewmodel/messages/MessageListViewModel.kt

```diff
@@ -236,6 +252,93 @@ public class MessageListViewModel(
         }
     }
 
+    /**
+     * Counts how many messages the user hasn't read already. This is based on what the last message they've seen is,
+     * and the current message state.
+     *
+     * @param newMessageState - The state that tells us if there are new messages in the list.
+     * @return - [Int] which describes how many messages come after the last message we've seen in the list.
+     * */
+    private fun getUnreadMessageCount(newMessageState: NewMessageState? = currentMessagesState.newMessageState): Int {
+        if (newMessageState == null || newMessageState == MyOwn) return 0
```

Comment:
since 0 is hardcoded , i feel we should declare const  immutable variable and use that, gets easy on code review and further logic changes to other parts of code in this function.

Please ignore if you feel otherwise.

Reply:
Honestly I don't think we should hardcoded a constant in this case, especially because it seems pretty clear what the 0 represents.

We're fetching the "unreadMessageCount", so if we return zero, then it's reasonable that there are no messages that are unread, right?

It is a pretty straightforward piece of logic in this case and I don't think a constant named e.g. `NO_UNREAD_MESSAGES` would make much of a difference.